package.json

```diff
@@ -23,6 +23,7 @@
     "@actions/core": "^1.10.1",
     "@googleapis/androidpublisher": "^12.1.0",
     "fast-glob": "^3.3.2",
+    "google-auth-library": "^9.15.0",
```

Comment:
❓ Did I miss why this was needed?

Reply:
Both the ecma target and this are related to ncc, so most likely a problem with that.